app/src/main/java/org/fossasia/openevent/app/utils/DateService.java

```diff
@@ -0,0 +1,53 @@
+package org.fossasia.openevent.app.utils;
+
+import org.fossasia.openevent.app.data.models.Event;
+
+import java.text.ParseException;
+import java.util.Date;
+
+public class DateService {
```

Comment:
It's not true. Larger the class gets, the messier it is. See the composition of standard libraries in Java, almost all of them have just method names in their class and everything is delegated to specific objects.

https://github.com/ReactiveX/RxJava/blob/2.x/src/main/java/io/reactivex/Observable.java

And this is not something out of the blue I am advocating, this is Facade Design Pattern

https://en.wikipedia.org/wiki/Facade_pattern

Distributing code among classes is opposite of making things messy. As this is a model class, it already pretty huge, or else, there should be a simple rule, break the code if it exceeds 200 lines